server/models/ConfigModel/index.js

```diff
@@ -1,22 +1,80 @@
+import { ObjectId } from 'mongodb'
 import defaultUserConfig from '../../configs/defaultUserConfig'
 import { collections } from '../../utils/mongoCollections'
 
 const ConfigModel = {
+  destroy: async (db, configId) => {
+    return await db
+      .collection(collections.configs)
+      .deleteOne({ _id: new ObjectId(configId) })
+  },
+  edit: async (db, configId, configAttributes) => {
+    console.log(configId, configAttributes)
+    return await db
+      .collection(collections.configs)
+      .findOneAndUpdate(
+        { _id: new ObjectId(configId) },
+        { $set: configAttributes },
+        { returnOriginal: false }
+      )
+  },
+  index: async (db, userId) => {
+    return await db
+      .collection(collections.configs)
+      .aggregate(ConfigModel.loadAllConfigurationsMongoQuery(userId))
+      .toArray()
+  },
+  loadAllConfigurationsMongoQuery: (userId) => {
```

Comment:
do we need this to be a key on the `ConfigModel` or can it be a "private" method above this `ConfigModel` constant that is used by the model methods? We should try to limit the APIs of these objects as much as we can so they are used as expected.

Reply:
[f11e811](https://github.com/AMP-SCZ/dpdash/pull/305/commits/f11e811a37bb7df8cc158a8d0e061615e8ec535d)